lint/config.go

```diff
@@ -12,13 +12,23 @@ type RuleConfig struct {
 // RulesConfig defines the config for all rules.
 type RulesConfig = map[string]RuleConfig
 
+// DirectiveConfig is type used for the linter directive configuration.
+type DirectiveConfig struct {
+	Severity  Severity
+}
+
+// DirectivesConfig defines the config for all directives.
+type DirectivesConfig = map[string]DirectiveConfig
+
+
 // Config defines the config of the linter.
 type Config struct {
-	IgnoreGeneratedHeader  bool `toml:"ignoreGeneratedHeader"`
+	IgnoreGeneratedHeader  bool             `toml:"ignoreGeneratedHeader"`
 	Confidence             float64
 	Severity               Severity
-	Rules                  RulesConfig `toml:"rule"`
-	ErrorCode              int         `toml:"errorCode"`
-	WarningCode            int         `toml:"warningCode"`
-	SpecifyDisableReason   bool `toml:"specifyDisableReason"`
+	Rules                  RulesConfig      `toml:"rule"`
+	ErrorCode              int              `toml:"errorCode"`
+	WarningCode            int              `toml:"warningCode"`
+	Directives             DirectivesConfig `toml:"directive"`
```

Comment:
How can we disable it?

Reply:
If `[directive.specify-disable-reason]` is not present in the config file ==> `Directives` is empty ==> it is disabled (see my previous comment above)